src/modules/commander/Commander.cpp

```diff
@@ -1610,6 +1614,18 @@ Commander::run()
 			}
 		}
 
+		if (orb_exists(ORB_ID(esc_status), 0) == PX4_OK) {
```

Comment:
Do you really need this? Is it not enough to check if it's updated and the timestamp is valid?

Reply:
You are right, this is a left over that I introduced then the logic was still reasoning on "is valid" and thus i needed to initialize that to true in case of escs not reporting status. Changing the message to `condition_escs_error` (and thus inverting the logic) automatically solved this corner case. Will get rid of that